template/template.yaml

```diff
@@ -46,11 +49,69 @@ options:
 
   - !Option
     name: probe-rs
-    display_name: Flashes using `probe-rs` instead of `espflash`.
-    help: "espflash uses a serial connection (UART) for programming and logging.\nprobe-rs is a
-      debugger that connects to the chips over JTAG. It can be used to flash and monitor, and
-      it can also be used to interactively debug an application, or run tests on the hardware.
-      Semihosting or RTT-based technologies like defmt-rtt require probe-rs."
+    display_name: Use probe-rs to flash and monitor instead of espflash.
+    help: probe-rs is a debugger that connects to the chips over JTAG. It can be used to flash and
+      monitor, and it can also be used to interactively debug an application, or run tests on the
+      hardware. Semihosting or RTT-based technologies like defmt-rtt require probe-rs.
+    chips:
+      - esp32c6
+      - esp32h2
+      - esp32s3
```

Comment:
C3 can also come with built-in jtag (?), not sure where it should live

Reply:
I'm not 100% sure, if the default state of the C3 is that the usb jtag is available, then I can move it. But the c3 devkits come with only one USB port that is a serial port. Since this grouping just prints more info, and since I'm unsure, I think the C3 should be good where it is.